config/core/test-configs.yaml

```diff
@@ -2986,6 +2986,13 @@ test_configs:
   - device_type: mt8365-evk
```

Comment:
This device type already exists, this is expanding the test set for the existing device rather than adding a new device.